Tutorials/start/telemetrystart.md

```diff
@@ -1,16 +1,36 @@
-#Starting With Telemetry
+# Starting With Telemetry
 
-Now that Faraday is configured and we know Proxy is running well we can view telemetry directly in the web browser for a quick diagnostic that everything is well. Also, if you're like us at FaradayRF we think data can be beautiful :) The [Telemetry](../../Applications/Telemetry) application is located in `Applications/Telemetry` is a Flask-based server application which saves data to a SQLite database. It then exposes a RESTful API to query the database.
+Now that Faraday is configured and we know Proxy is running well we can view telemetry directly in the web browser for a quick diagnostic that everything is well. Also, if you're like us at FaradayRF we think data can be beautiful :) The [Telemetry](../../Applications/Telemetry) application is a Flask-based server which saves data to a SQLite database. It then exposes a RESTful API to query the database.
 
-Since [Telemetry](../../Applications/Telemetry) responds with JSON data we could use a web browser to print it to the screen, Python to interact with it, or an application specific to HTTP communications to interact.
+Since [Telemetry](../../Applications/Telemetry) responds with JSON data we could use a web browser to print it to the screen, Python, or any application able to query JSON responses with HTTP communications to interact.
+
+## Initializing Telemetry
+Just as we've done many times, we need to initialize the  `faraday-telemetry` configuration file. Simply run:
+* `faraday-telemetry --init-config`
+
+![Telemetry Configuration](images/faraday-telemetry-init.jpg)
 
 ## Configuring & Running Telemetry
 
-The configuration guide in [Telemetry](../../Applications/Telemetry) documentation sums up how to get the application up and running. We suggest you head over to that documentation to step through configuration. Once setup, play around with the example queries using your web browser located on the bottom of the documentation. Report back here when complete!
+To configure the `faraday-telemetry` application simply tell it what the Proxy callsign and nodeid is as programmed when we configured `faraday-proxy`:
+
+* `faraday-telemetry --callsign kb1lqc --nodeid 2`
```

Comment:
ACTION @kb1lqc : DOH! I started by using a fresh install of pip (manually deleted the python27 etc folder contents. I got this error because the installation does not install telemetrydb.sql.

![image](https://user-images.githubusercontent.com/339144/27819576-c77ad662-604f-11e7-9a94-ed8e3a945273.png)

![image](https://user-images.githubusercontent.com/339144/27819583-ce00fe44-604f-11e7-940b-3fdc22f941f3.png)

As shown BELOW the GITHUB repo clearly has the correct file:
![image](https://user-images.githubusercontent.com/339144/27819655-11dfc622-6050-11e7-9bfc-b31d3c4e2b89.png)